compact/far_backward/magnets.xml

```diff
@@ -13,33 +13,40 @@
     type="BeamPipeChain"
     wall_thickness="2*mm"
     vis="BeamPipeVis">
-      <pipe id="0" name="Pipe_to_Q1eR"
-        xcenter="0" zcenter="(Center_Beampipe_End + Q1eR_CenterPosition+Q1eR_Length/2)/2"
-        length="Center_Beampipe_End - (Q1eR_CenterPosition+Q1eR_Length/2)" theta="0"
-        rout1="Center_Beampipe_Rad" rout2="Q1eR_InnerRadius">
+      <pipe id="0" name="Pipe_0"
+        xcenter="0" zcenter="-7342.0 * mm + (Center_Beampipe_End + 7342.0 * mm)/2"
+        length="Center_Beampipe_End + 7342.0 * mm" theta="0"
+        rout1="Center_Beampipe_Rad" rout2="(111.40/2 + 2) * mm">
```

Comment:
It is true. However, I would suggest keeping the beam pipe geometry separated from the magnet geometry because they have different sources (i.e., the beam pipe model is from the vacuum and background groups, magnets -- lattice and magnet groups). Also, the inner radius of magnets should not always follow the beam pipe radius. 
I can create beam pipe global variables in the central-beam pipe XML file to be used here. What do you think @wdconinc ?

Reply:
I am not sure if we need them all as global variables in the central beam pipe; they can be local variables in the far_backward and far_forward sections, right?